src/app-layer-smb.h

```diff
@@ -15,6 +15,12 @@
  * 02110-1301, USA.
  */
 
+/**
+ * \file 
+ * 
+ * Header file for SMB protocol handling
```

Comment:
no need to state that this is a header file, the .h extension already tells us that

Reply:
Is it okay to leave it as SMB for protocol handling?

Reply:
I ran it by Juliani and she thought it better to have header file included in the naming. 

Kindly advice on how to go about it.